.gitignore

```diff
@@ -31,3 +31,6 @@ firebase
 firebase.json
 .firebase/
 src/purifier/dist
+extensions/amp-analytics/0.1/vendors/*.json
```

Comment:
This was removed in https://github.com/ampproject/amphtml/commit/2f76bee2ebaa5675350d5e39bd6d06982940f57a